package.json

```diff
@@ -15,12 +15,15 @@
     "tables",
     "mysql"
   ],
-  "author": "Wikimedia Service Team <services@wikimedia.org>",
+  "contributors": [
+    "Wikimedia Service Team <services@wikimedia.org>",
+    "Femiwiki Team <admin@femiwiki.com> (https://femiwiki.com/)"
+  ],
   "license": "Apache-2.0",
   "bugs": {
-    "url": "https://phabricator.wikimedia.org/tag/restbase/"
+    "url": "https://github.com/wikimedia/restbase-mod-table-mysql/issues"
```

Comment:
A discussion for where is more suitable for this project is needed. Using Phabricator gives developers and volunteers more opportunities to find issues.

Reply:
Phabricator is the standard place for issues and bug for all Wikimedia-owned repositories, so it'll have to stay that way.

Reply:
@simnalamburt: Can I fix this myself If you do not have enough time?

Reply:
OK it's all yours